src/base/net/portforwarder.h

```diff
@@ -37,8 +37,7 @@ namespace Net
         Q_DISABLE_COPY_MOVE(PortForwarder)
 
     public:
-        explicit PortForwarder(QObject *parent = nullptr);
-        ~PortForwarder() override;
+        virtual ~PortForwarder() override;
```

Comment:
👎 
This was already discussed at the dawn of c++11. Since `override` implies `virtual` we prohibit to use both at the same time.